apps/sms/views/shared/test/unit/utils_test.js

```diff
@@ -1021,14 +1021,46 @@ suite('Utils', function() {
     });
   });
 
-  suite('Utils.params', function() {
+  suite('URL utils', function() {
```

Comment:
__unrelated note:__ this suite name makes me think that we need smth like ```Utils.URL...``` namespace as we did for ```Utils.Promise...```, but not now obviously :)